configure.sh

```diff
@@ -137,7 +137,10 @@ case $(uname -s) in
         cc_is_clang || die "Only 'clang' is supported on OpenBSD"
         [ "${TARGET_ARCH}" = "x86_64" ] ||
             die "Only 'x86_64' is supported on OpenBSD"
-        ld_is_lld || die "Only 'ld.lld' is supported on OpenBSD"
+        if [ !ld_is_lld ]; then
+            LD='/usr/bin/ld.lld'
+            echo "WARNING: using 'ld.lld' as 'ld' is not supported on OpenBSD"
```

Comment:
This should include a check that `/usr/bin/ld.lld` exists and is executable, and `die` if not. This is so that users on OpenBSD systems without `ld.lld` are not confused when things start failing mysteriously further down the chain.

Reply:
I am very sure any OpenBSD 6.2 or later with have ld.lld, as it is part of base, but there is no harm in having the check.

I have added this to configure.sh.